From 54b8297f072b3882700f5426478fe2700bc733fd Mon Sep 17 00:00:00 2001
From: Adam Cohen Hillel <adamcohenhillel@gmail.com>
Date: Wed, 27 May 2026 00:19:20 +0300
Subject: [PATCH 1/4] OpenPhone: fix debuggerd host build on Darwin

Change-Id: I2a385fa8101222bf1caa4444f812dd6f410c6409

diff --git a/debuggerd/libdebuggerd/utility_host.cpp b/debuggerd/libdebuggerd/utility_host.cpp
index f2354582a..3c1687e41 100644
--- a/debuggerd/libdebuggerd/utility_host.cpp
+++ b/debuggerd/libdebuggerd/utility_host.cpp
@@ -17,7 +17,9 @@
 #include "libdebuggerd/utility_host.h"
 
 #include <ctype.h>
+#if defined(__linux__)
 #include <sys/prctl.h>
+#endif
 
 #include <charconv>
 #include <limits>
-- 
2.50.1 (Apple Git-155)

