From 0ced767926b1b7f7f7be2df72e5ad56880833b28 Mon Sep 17 00:00:00 2001
From: Adam Cohen Hillel <adamcohenhillel@gmail.com>
Date: Thu, 28 May 2026 14:40:03 +0300
Subject: [PATCH 3/4] OpenPhone: define MTE signal codes for Darwin host builds

Change-Id: I16feecf775aa455c677b7c7e03371c57a16d15a6

diff --git a/debuggerd/libdebuggerd/tombstone_proto_to_text.cpp b/debuggerd/libdebuggerd/tombstone_proto_to_text.cpp
index 438fb9f4a..d6bc5cf9e 100644
--- a/debuggerd/libdebuggerd/tombstone_proto_to_text.cpp
+++ b/debuggerd/libdebuggerd/tombstone_proto_to_text.cpp
@@ -37,6 +37,14 @@
 #include "libdebuggerd/utility_host.h"
 #include "tombstone.pb.h"
 
+#ifndef SEGV_MTEAERR
+#define SEGV_MTEAERR 8
+#endif
+
+#ifndef SEGV_MTESERR
+#define SEGV_MTESERR 9
+#endif
+
 using android::base::StringAppendF;
 using android::base::StringPrintf;
 
-- 
2.50.1 (Apple Git-155)

