From ed4a341deaaeec708f3b6acf124b00ab7f36f3ec Mon Sep 17 00:00:00 2001
From: Adam Cohen Hillel <adamcohenhillel@gmail.com>
Date: Thu, 28 May 2026 14:45:25 +0300
Subject: [PATCH 4/4] OpenPhone: link pbtombstone with CoreFoundation on Darwin

Change-Id: I70d2bd7bd7c1b2627dc1edff0699b012f7d9a0dc

diff --git a/debuggerd/Android.bp b/debuggerd/Android.bp
index fe2b055de..b5a6a7aae 100644
--- a/debuggerd/Android.bp
+++ b/debuggerd/Android.bp
@@ -345,6 +345,11 @@ cc_binary {
         "libprotobuf-cpp-lite-notls-for-linker",
         "libtombstone_proto",
     ],
+    target: {
+        darwin: {
+            host_ldlibs: ["-framework CoreFoundation"],
+        },
+    },
 }
 
 cc_test_library {
-- 
2.50.1 (Apple Git-155)

