From 3fc90b66be3347a034316405376cef1662e3b7dd Mon Sep 17 00:00:00 2001
From: Adam Cohen Hillel <adamcohenhillel@gmail.com>
Date: Thu, 28 May 2026 14:49:05 +0300
Subject: [PATCH 3/3] OpenPhone: link Darwin host tools with CoreFoundation

Change-Id: I8c95b5e6af8afac160203a92da8fb28bc3cf65ee

diff --git a/cc/config/darwin_host.go b/cc/config/darwin_host.go
index 982c2aeb3..d343e92fb 100644
--- a/cc/config/darwin_host.go
+++ b/cc/config/darwin_host.go
@@ -47,6 +47,7 @@ var (
 		"-mmacosx-version-min=${macMinVersion}",
 		"-m64",
 		"-mlinker-version=305",
+		"-framework CoreFoundation",
 	}
 
 	darwinSupportedSdkVersions = []string{
-- 
2.50.1 (Apple Git-155)

